.github/workflows/release.yaml

```diff
@@ -6,7 +6,6 @@ on:
       - main
     paths:
       - 'charts/**'
-      - '!charts/datadog-csi-driver/**'
```

Comment:
Can we keep this here and let the csi-driver chart owners remove this when they're ready to publish?

Reply:
thanks for catching this, this wasn't intentional, started on stale branch and copied.